pkg/k8sports/kubernetes_helpers.go

```diff
@@ -1,3 +1,4 @@
+// Copyright 2015 The Kubernetes Authors.
 // Copyright (c) 2017 Intel Corporation
```

Comment:
Also IMO you should retain full Kubernetes header (with license) and then post Intel header with notice of modification.